CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+### citus v10.2.0 (September 11, 2021) ###
+
+* Adds PostgreSQL 14 support
+
+* Adds hash & btree index support for columnar tables
+
+* Adds helper UDFs for easy time partition management:
+  `get_missing_time_partition_ranges`,
+  `create_time_partitions`,
+  `drop_old_time_partitions`
+
+* Adds a parameter to cleanup metadata when metadata syncing is stopped
```

Comment:
minor: I'd move this as the last item with `Adds ..`

Reply:
moved at the end, after the `Fixes ..` items